README.md

```diff
@@ -23,52 +23,52 @@ Official website: https://parity.io | Be sure to check out [our wiki](https://wi
 
-If you run into an issue while using Parity, feel free to file one in this repository or hop on our [Gitter](https://gitter.im/paritytech/parity) or [Riot](https://riot.im/app/#/group/+parity:matrix.parity.io) chat room to ask a question. We are glad to help! **For security-critical issues**, please refer to the security policy outlined in [SECURITY.MD](SECURITY.md).
+If you run into an issue while using Parity-Ethereum, feel free to file one in this repository or hop on our [Gitter](https://gitter.im/paritytech/parity) or [Riot](https://riot.im/app/#/group/+parity:matrix.parity.io) chat room to ask a question. We are glad to help! **For security-critical issues**, please refer to the security policy outlined in [SECURITY.MD](SECURITY.md).
 
-Parity's current beta-release is 1.11. You can download it at https://github.com/paritytech/parity/releases or follow the instructions below to build from source.
+Parity-Ethereum's current beta-release is 2.0. You can download it at https://github.com/paritytech/parity/releases or follow the instructions below to build from source.
 
 ----
 
 ## Build dependencies
 
-**Parity requires Rust version 1.26.0 to build**
+**Parity-Ethereum requires Rust version 1.27.0 to build**
 
 We recommend installing Rust through [rustup](https://www.rustup.rs/). If you don't already have rustup, you can install it like this:
 
 - Linux:
-	```bash
-	$ curl https://sh.rustup.rs -sSf | sh
-	```
+  ```bash
+  $ curl https://sh.rustup.rs -sSf | sh
+  ```
 
-	Parity also requires `gcc`, `g++`, `libssl-dev`/`openssl`, `libudev-dev`, `pkg-config`, `file` and `make` packages to be installed.
+  Parity-Ethereum also requires `gcc`, `g++`, `libudev-dev`, `pkg-config`, `file`, `make`, and `cmake` packages to be installed.
 
 - OSX:
-	```bash
-	$ curl https://sh.rustup.rs -sSf | sh
-	```
+  ```bash
+  $ curl https://sh.rustup.rs -sSf | sh
+  ```
 
-	`clang` is required. It comes with Xcode command line tools or can be installed with homebrew.
+  `clang` is required. It comes with Xcode command line tools or can be installed with homebrew.
 
 - Windows
   Make sure you have Visual Studio 2015 with C++ support installed. Next, download and run the rustup installer from
-	https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe, start "VS2015 x64 Native Tools Command Prompt", and use the following command to install and set up the msvc toolchain:
+  https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe, start "VS2015 x64 Native Tools Command Prompt", and use the following command to install and set up the msvc toolchain:
   ```bash
-	$ rustup default stable-x86_64-pc-windows-msvc
+  $ rustup default stable-x86_64-pc-windows-msvc
   ```
 
 Once you have rustup installed, then you need to install:
 * [Perl](https://www.perl.org)
 * [Yasm](http://yasm.tortall.net)
```

Comment:
Is this true? I have not run into any assembly code in the code base, perhaps this is stale info?

Reply:
I have no clue about building on Windows.

Reply:
[ring](https://github.com/briansmith/ring) required `perl` and `ASM` to generate build some crypto-modules written in perl/ASM. So I think it is still true if nothing has changed in our ```fork of ring```

Reply:
I see, thanks.